recipe-client-addon/jar.mn

```diff
@@ -7,6 +7,7 @@
   lib/ (./lib/*)
   data/ (./data/*)
   skin/  (skin/*)
-  vendor/ (./vendor/*)
 % resource shield-recipe-client-content %content/
   content/ (./content/*)
+% resource shield-recipe-client-vendor %vendor/
```

Comment:
I wonder, does it work as-is if you add `contentaccessible=yes` flags to the end? If so, might as well do that here in preparation for bug 863246.